Resources/Private/Php/Fetch/src/Fetch/Message.php

```diff
@@ -543,7 +543,7 @@ protected function processStructure($structure, $partIdentifier = null)
                 }
             }
 
-            if (strtolower($structure->subtype) === 'plain' || ($structure->type == 1 && strtolower($structure->subtype) !== 'alternative')) {
+            if ((strtolower($structure->subtype) === 'plain') || ($structure->type == 1 && strtolower($structure->subtype) !== 'alternative') || ($structure->type == 0 && strtolower($structure->subtype) == 'rfc822-headers')) {
```

Comment:
The Fetch sourcecode which is used to read the email has its own repo at https://github.com/tedious/Fetch/blob/master/src/Fetch/Message.php#L546

Although there seems not to be much activity in that project the change to Fetch should be made upstream by adding a pull request there.

If its not merged promptly (and Fetch can be updated here) at least a comment this is a modified version should be added